lib/xml2.xml

```diff
@@ -12,6 +12,12 @@ metalanguage for markup languages.
 
 This package provides a library providing an extensive API to handle
 such XML data files.</description>
+  <icon href="https://raw.githubusercontent.com/0install/repo.roscidus.com/master/lib/xml2.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="https://raw.githubusercontent.com/0install/repo.roscidus.com/master/lib/xml2.png" type="image/png"/>
+  <category>Development</category>
+  <homepage>http://gnuwin32.sourceforge.net/packages/libxml2.htm</homepage>
```

Comment:
I'd suggest not putting the gnuwin32 homepage here, since this feed also contains Linux builds.

Reply:
updated to http://xmlsoft.org/